src/notification/index.js

```diff
@@ -234,7 +221,7 @@ export class NotificationListener {
 
   /** Private. */
   handleIOSRegistrationFailure = (err: NotificationRegistrationFailedEvent) => {
-    logging.warn(`Failed to register iOS push token: ${err.domain}:#${err.code}`, {
+    logging.warn(`Failed to register iOS push token: ${err.message}`, {
```

Comment:
> Let's be sure to keep the code

Makes sense. I think I was counting on either the code being part of `message` (along with the domain), or that `message` was composed in some way I don't really understand, but that Apple is confident will describe the error well. From the [doc on `localizedDescription`](https://developer.apple.com/documentation/foundation/nserror/1414418-localizeddescription?language=objc):

```
The object in the user info dictionary for the key NSLocalizedDescriptionKey. If the user info dictionary doesn’t contain a value for NSLocalizedDescriptionKey, a default string is constructed from the domain and code.
```

It totally makes sense to at least keep the code by itself in the first argument to `logging.warn`, since they give it to us.